backend/test_app/test_account_endpoints.py

```diff
@@ -149,6 +149,26 @@ def test_update_account(client, test_user):
 #  -------------------------------------------------------------------
 
 
+def test_account_remove(client, test_user):
+    """
+    Account remove by id endpoint test
+    """
+    access_token = test_setup_login_user(client, test_user)
+    account_response = client.get(
+        "/peppermint/account/my_accounts",
+        headers={"Authorization": f"Bearer {access_token}"},
+    )
+    account = account_response.json()[0]
+    account_id = account["id"]
+
+    response = client.delete(
+        f"/peppermint/account/{account_id}",
+        headers={"Authorization": f"Bearer {access_token}"},
+    )
+
+    assert response.status_code == 204
+
```

Comment:
I would also add a test that checks the account itself is actually deleted and the account is not showing up when you query a user.